test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.client.cs

```diff
@@ -123,6 +123,207 @@ public void GrpcClientCallsAreCollectedSuccessfully(string baseAddress, bool sho
             }
         }
 
+        [Theory]
+        [InlineData("http://localhost")]
+        [InlineData("http://localhost", false)]
+        [InlineData("http://127.0.0.1")]
+        [InlineData("http://127.0.0.1", false)]
+        [InlineData("http://[::1]")]
+        [InlineData("http://[::1]", false)]
+        public void GrpcClientCallsAreCollectedSuccessfully_New(string baseAddress, bool shouldEnrich = true)
+        {
+            try
+            {
+                Environment.SetEnvironmentVariable("OTEL_SEMCONV_STABILITY_OPT_IN", "http");
```

Comment:
Can you use the IConfiguration here instead of `SetEnvironmentVariable`? 
That will help prevent test conflicts.

Reply:
done